upcoming/ssh-server-in-go.md

```diff
@@ -0,0 +1,240 @@
++++
+author = ["Jiahua Chen"]
+date = "2015-12-18T04:07:33-05:00"
+title = "Writing SSH server in Go"
```

Comment:
Consider putting _an_ in front of _SSH_ so it reads more clearly. _"Writing an SSH server in Go"_